Extensions/PrimitiveDrawing/JsExtension.cpp

```diff
@@ -26,7 +26,9 @@ class PrimitiveDrawingJsExtension : public gd::PlatformExtension {
     GetObjectMetadata("PrimitiveDrawing::Drawer")
         .SetIncludeFile(
             "Extensions/PrimitiveDrawing/shapepainterruntimeobject.js")
-        .AddIncludeFile("Extensions/PrimitiveDrawing/pixi-graphics-extras/graphics-extras.min.js")
+        .AddIncludeFile(
+            "Extensions/PrimitiveDrawing/pixi-graphics-extras/"
+            "graphics-extras.min.js")
```

Comment:
Auto-formatting the code that is not actually changed make the review a lot harder and the risk of missing a change to review higher.